src/test/scala/uk/gov/nationalarchives/consignmentexport/BagMetadataSpec.scala

```diff
@@ -75,4 +76,18 @@ class BagMetadataSpec extends ExportSpec {
     }
     exception.getMessage should equal(s"Incomplete details for user $userId")
   }
+
+  //Judgments don't have a series. So will be set to empty for export to succeed
```

Comment:
I would delete this comment, and change to test name slightly to: "_.... empty series id for a 'judgment' consignment type_"